src/main/java/tictactoe/Board.java

```diff
@@ -4,10 +4,15 @@
 
 public class Board implements IBoard {
     private Marks[][] board;
-    private final int SIZE = 3;
+    private int size = 3;
 
     public Board() {
-        this.board = new Marks[SIZE][SIZE];
+        this.board = new Marks[size][size];
```

Comment:
Now with the parameterised constructor available I think you can just delegate to that one by `this(3)`.